JamesIves/fetch-api-data-action
# Fetch Retry Requests

**Description**
> Provide a description of what your changes do.

Adds the ability to pass in `RETRY: true` to the configuration, making any failed requests retry before failing. If set to true the requests will be attempted  one time, and then retried three more times. This will occur on the token and default endpoint requests until the job either fails or completes.

Also adds cross publishing support to the GitHub registry within `publish.yml`. 

**Testing Instructions**
> Give us step by step instructions on how to test your changes.

* Set `RETRY` to `true`.
* Attempt to run a failed API request: You can see some failed logs here (The logging has since been cleaned up): https://github.com/JamesIves/fetch-api-data-action/runs/552234486?check_suite_focus=true

**Additional Notes**
> Anything else that will help us test the pull request.

* I've also switched `console.log` to `info` as it's part of the actions toolkit suite.
